src/py42/sdk/queries/alerts/alert_query.py

```diff
@@ -36,3 +36,17 @@ def __str__(self):
             self.sort_key,
         )
         return json
+
+    def __iter__(self):
```

Comment:
I also added `__iter__` functions to `AlertQuery`, `FileEventQuery`, `FilterGroup`, and `QueryFilter`. This allows `dict(obj)` to return a dictionary representation of the JSON version of those objects easily.